src/settings/components/general-settings/user-quota.tsx

```diff
@@ -58,7 +58,7 @@ const UserQuota: FC<UserQuotaProps> = ({ mobileView }) => {
 	return mobileView ? (
 		<Container width="fit" padding={{ right: 'medium' }}>
 			<Tooltip label={description} placement="bottom">
-				<Quota fill={quota > 0 ? quota : 0} fillBackground={fillBackground} />
+				<Quota fill={quota >= 100 ? 100 : quota} fillBackground={fillBackground} />
```

Comment:
This logic does not correspond to the one below. Here you are not considering the -1 value.
I think we should merge the two conditionals (this one and the one of line 79) in a single one, and use the resulting const in the props.
Check if the quota can have value -1 or not before deciding which of the two logic we need to keep

Reply:
yep, seems good now